test/functional/apps/discover/esql/_esql_view.ts

```diff
@@ -253,7 +253,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.discover.selectTextBaseLang();
         await PageObjects.header.waitUntilLoadingHasFinished();
         await PageObjects.discover.waitUntilSearchingHasFinished();
-        await PageObjects.discover.selectIndexPattern('logstash-*', false);
+        await testSubjects.click('esql-menu-button');
+        await testSubjects.click('switch-to-dataviews');
```

Comment:
Can we replace these new clicks in the modified functional tests with `switchToDataViewMode()` call?